README.md

```diff
@@ -204,7 +204,8 @@ The above statement is also valid for all gateway backends.
 ## Test using MinIO Console
 MinIO Server comes with an embedded web based object browser. Point your web browser to http://127.0.0.1:9000 to ensure your server has started successfully.
 
-> NOTE: MinIO runs console on random port by default if you wish choose a specific port use `--console-address` to pick a specific interface and port.
+> NOTE: MinIO runs console on random port by default if you wish choose a specific port use `--console-address` to pick a specific interface and port. MinIO redirects browser access requests http://127.0.0.1:9000 to console port, please make sure console port is accessible by choosing a specific port `--console-address`. You can also choose to override this by setting `MINIO_BROWSER_REDIRECT_URL` environment variable if you are running MinIO with a frontend proxy or ingress rule.
```

Comment:
```suggestion
> NOTE: MinIO runs console on random port by default if you wish choose a specific port use `--console-address` to pick a specific interface and port. 

MinIO redirects browser access requests http://127.0.0.1:9000 to the console port, regardless of it being set explicitly or dynamically. MinIO uses the host IP address as the redirect URL. This URL and port *must* be accessible by the client for the redirection to work.

For deployments behind a load balancer, proxy, or ingress rule where the MinIO host IP address is not public, use the `MINIO_BROWSER_REDIRECT_URI` environment variable to specify the external hostname for the redirect. The LB/Proxy must have rules for directing traffic to the Console port specifically.

For example, consider a MinIO deployment behind a proxy `https://minio.example.net` with rules for forwarding traffic on port :9000 and :9001 to  MinIO and the MinIO Console respectively on the internal network. Set `MINIO_BROWSER_REDIRECT_URL` to `https://minio.example.net:9001` to ensure the browser receives a valid reachable URL.

```

Minor followup-  do we always redirect to localhost, or do we redirect based on the resolved IP for the incoming request? i.e. if it was `minio.example.net -> 192.0.2.10`, would we redirect to `192.0.2.10:CONSOLEPORT` ?

Reply:
We resolve based on the incoming request @ravindk89